images/build/GitHubActions.Dockerfile

```diff
@@ -0,0 +1,165 @@
+# Folders in the image which we use to build this image itself
+# These are deleted in the final stage of the build
+ARG IMAGES_DIR=/tmp/oryx/images
+ARG BUILD_DIR=/tmp/oryx/build
+ARG SDK_STORAGE_ENV_NAME
+ARG SDK_STORAGE_BASE_URL_VALUE
+# Determine where the image is getting built (DevOps agents or local)
+ARG AGENTBUILD
+
+# Use the exact layer that is present on the GitHub agent machines
+# https://github.com/actions/virtual-environments/blob/master/images/linux/Ubuntu1804-README.md
+FROM buildpack-deps:stretch@sha256:8bcd320ec29cf67052985f28891586fb853051f69ad0646fc7a49f47d6e3ee1a AS main
```

Comment:
@kichalla I'm not sure how often GitHub is updating their runners with a new SHA of this image, so my only concern would be keeping this Dockerfile in sync with their cache, and what it means to our pull times if it becomes out of sync.

Reply:
Yes, that is a cause of concern. We have to work with DevOps team to make sure both of our team's base images are aligned.